build/Dockerfile.operator

```diff
@@ -1,6 +1,5 @@
-ARG BASE_IMAGE=quay.io/nmstate/kubernetes-nmstate
-ARG BASE_TAG=latest
-FROM ${BASE_IMAGE}:${BASE_TAG} AS handler
+ARG BASE_IMAGE=quay.io/nmstate/kubernetes-nmstate-handler:latest
```

Comment:
Why do we need this change here?

Reply:
To compose the image name at only one place, it's connected with release.